.github/functional_tests/reinstall-transient-toolchain/action.yml

```diff
@@ -0,0 +1,70 @@
+name: 'Reinstall Transient Toolchain Functional Test'
+description: |
+  Run `lean-action` with `reinstall-transient-toolchain: true`.
+  Verify that the toolchain will be redownloaded.
+inputs:
+  toolchain:
+    description: 'Toolchain to use for `lake init` (not the toolchain that will be tested for reinstallation).'
+    required: true
+runs: 
+  using: 'composite'
+  steps:
+    # TODO: once `lean-action` supports just setup, use it here
```

Comment:
Isn't that possible now, given that mathlib uses this? (Or should the comment refer to something else?)

Reply:
The problem, I believe, is that the setup step still needs a lake package (`lean-toolchain` file to be precise). So this is still relevant to the tests.